### PR TITLE
created entities

### PR DESCRIPTION
**Pull request by robertzthomas**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,5 @@
 # IDEA files
 *.idea
 
+.classpath
+.project
```
Comment (gryclmn) [inline .gitignore:14]: Thanks for adding these Nick!

File: out/production/Final Project/.gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/application/
+/dao/
+/entity/
+/utility/
```
Comment (gryclmn) [inline out/production/Final Project/.gitignore:4]: Not sure where this file came from? Possibly auto-made by Eclipse?

File: src/entity/Dish.java [hunk 3]
```diff
@@ -1,4 +1,27 @@
...
+	}
+
+	public String getDishName() {
+		return dishName;
+	}
+
+	public void setDishName(String dishName) {
+		this.dishName = dishName;
+	}
 }
```
Comment (gryclmn) [inline src/entity/Dish.java:27]: Looking good!

File: src/entity/Meal.java [hunk 4]
```diff
@@ -1,4 +1,53 @@
...
+	}
+	public MealType getType() {
+		return type;
+	}
+	public void setType(MealType type) {
+		this.type = type;
+	}
+	
+	
+}
```
Comment (gryclmn) [inline src/entity/Meal.java:52]: NIT: whitespace should be limited to one return for spacing.

File: src/entity/Dish.java (hunk 3 above)
Comment (robertzthomas) [reply src/entity/Dish.java:27]: thank you

**Comment by gryclmn** [comment]
LGTM. My turn. Go ahead and squash and merge. :)

**Pull request merged**